nerdlets/util/async.js

```diff
@@ -225,26 +225,26 @@ async function processBatchOfCardinalities(
   batchOfRules,
   accountId,
   errorCount,
-  maxVisibileErrors,
+  maxVisibileerror,
```

Comment:
It looks like you might have replaced `Errors` with error as well though this doesn't cause errors it changes the names of some of the variables

Reply:
otherwise looks mint

Reply:
Whoops! Good catch. Went ahead and made those fixes